lib/scrabble.rb

```diff
@@ -1,7 +1,33 @@
+require "pry"
 class Scrabble
 
+  attr_reader :word
+
+  def initialize
+    @word = word
+  end
+
+  def score
+    letters = word.split('')
+
+    total = 0
+    letters.each do |letter|
+      total += point_values[letter]
+    end
+    total
+  end
+
   def score(word)
```

Comment:
Have a second definition of score? Seems like this is overriding the first. Don't have two methods with the same name.